sale_delivery_state/models/sale_order.py

```diff
@@ -63,16 +79,23 @@ def _partially_delivered(self):
             for line in sale_lines
         )
 
-    @api.depends("order_line.qty_delivered", "state", "force_delivery_state")
+    @api.depends(
+        "order_line.qty_delivered",
+        "order_line.skip_sale_delivery_state",
+        "state",
+        "force_delivery_state",
+    )
     def _compute_oca_delivery_status(self):
+        # Loose dep on sale_stock
+        has_pickings = "picking_ids" in self._fields
```

Comment:
Well, I should find a better solution for this since the field is stored. Probably we need a sale_stock glue module if we want to support the "started" state. I'll add a TODO.